common/src/main/java/org/candlepin/common/config/PropertyConverter.java

```diff
@@ -104,7 +104,7 @@ protected static Number toNumber(Object value, Class<?> clazz) throws Conversion
             return (Number) value;
         }
         else {
-            String str = value.toString();
+            String str = value.toString().trim();
```

Comment:
Non-NACKing comment: Not a huge fan of the fact we aren't first checking if the value is null, but that unfortunate oversight predates this change.